scripts/xtensa-build-zephyr.py

```diff
@@ -173,42 +174,22 @@ def parse_args():
 						help="Path to a non-default rimage signing key.")
 	parser.add_argument("-o", "--overlay", type=pathlib.Path, required=False, action='append',
 						default=[], help="Paths to overlays")
-	parser.add_argument("-z", "--zephyr-ref", required=False,
-						help="Initial Zephyr git ref for the -c option."
-						" Can be a branch, tag, full SHA1 in a fork")
-	parser.add_argument("-u", "--url", required=False,
-						default="https://github.com/zephyrproject-rtos/zephyr/",
-						help="URL to clone Zephyr from")
 	mode_group = parser.add_mutually_exclusive_group()
-	mode_group.add_argument("-p", "--west_path", required=False, type=pathlib.Path,
-				help="""Points to existing Zephyr project directory. Incompatible with -c.
-Uses by default path used by -c mode: SOF_TOP/zephyrproject.
-If zephyr-project/modules/audio/sof is missing then a
-symbolic link pointing to SOF_TOP directory will automatically be
-created and west will recognize it as its new sof module.
-If zephyr-project/modules/audio/sof already exists and is a
-different copy than where this script is run from, then the
-behavior is undefined.
-This -p option is always required_if the real (not symbolic)
-sof/ and zephyr-project/ directories are not nested in one
-another.""",
-	)
-	mode_group.add_argument("-c", "--clone_mode", required=False, action="store_true",
-				help="""Using west, downloads inside this sof clone a new Zephyr
-project with the required git repos. Creates a
-sof/zephyrproject/modules/audio/sof symbolic link pointing
-back at this sof clone.
-Incompatible with -p. To stop after downloading Zephyr, do not
-pass any platform or cmake argument.""",
-	)
+	mode_group.add_argument("-i", "--init", required=False, action="store_true",
+		help="Initializes SOF submodules only without cloning Zephyr and its dependencies.")
+	mode_group.add_argument("-u", "--update", required=False, action="store_true",
+		help="""Runs west update command - downloads next to this sof clone a new Zephyr
+project with its required dependencies. Creates a modules/audio/sof symbolic link pointing
+back at this sof clone. Initializes SOF dependencies. All projects are checkout out to
+revision defined in manifests of SOF and Zephyr.""")
 	parser.add_argument('-v', '--verbose', default=0, action='count',
-			    help="""Verbosity level. Repetition of the flag increases verbosity.
+		help="""Verbosity level. Repetition of the flag increases verbosity.
 The same number of '-v' is passed to "west".
 """,
 	)
 	# Cannot use a standard -- delimiter because argparse deletes it.
 	parser.add_argument("-C", "--cmake-args", action='append', default=[],
-			    help="""Cmake arguments passed as is to cmake configure step.
+		help="""Cmake arguments passed as is to cmake configure step.
```

Comment:
Please no whitespace changes in this PR; it's not a small PR and it is _very_ disruptive for CI so review distractions and git conflicts are really best avoided.

Reply:
Yeah ok I will revert them, sorry.